### PR TITLE
Feature/encrypt keys ini

### PR DESCRIPTION
**Pull request by silverdaz**
`keys.ini` is:
* encrypted,
* not in cleartext on disk.

The keyserver loads the file and decrypts it (using `LEGA_PASSWORD`).
Decryption calls `openssl` as it is quicker than coding the AES decryption. Maybe simpler too.

### CODE REVIEW

File: deployments/docker/test/Makefile [hunk 1]
```diff
@@ -1,8 +1,9 @@
 .PHONY: upload submit user
 
 # folder for the localegarepo
-MAIN_REPO=~/LocalEGA
-# the keys need to be in ~/.ssh path
+MAIN_REPO=~/_ega
```
Comment (blankdots) [inline deployments/docker/test/Makefile]: It was like that `~/_ega` but, I would still keep `~/LocalEGA` to make it clear for everyone that is the LocalEGA repo and everyone that uses this test script changes accordingly to their path; I also am changing every time on my machine. 
Not a big deal, nitpicking :)

Comment (silverdaz) [reply deployments/docker/test/Makefile]: Yeah. My bad. Let's keep it as `~/LocalEGA`.

File: lega/conf/__init__.py [hunk 2]
```diff
@@ -159,5 +159,17 @@ def __init__(self, args=None, encoding='utf-8'):
...
-        self.read(conf_file, encoding=encoding)
+        filepath = Path(args[args.index('--keys') + 1]).expanduser()
+
+        if filepath.suffix != '.enc':
+            conf = filepath.open(encoding=encoding).read()
+        else:
+            # Quick solution, just for testing, to decrypt the encrypted keys configuration file (keys.ini)
+            # The 'LEGA_PASSWORD' must be defined
+            assert 'LEGA_PASSWORD' in os.environ, "LEGA_PASSWORD must be defined as an environment variable"
+            s = "openssl enc -aes-256-cbc -d -in {} -k {}".format(str(filepath), os.environ.get('LEGA_PASSWORD',None))
```
Comment (blankdots) [inline lega/conf/__init__.py]: Use solution provided by: https://github.com/NBISweden/LocalEGA/issues/325

**Review by blankdots — CHANGES_REQUESTED** [review]
I think we should have Keyserver configuration decrypted in code

**Review by blankdots — APPROVED** [review]

**Pull request merged**